### DISCUSSION TITLE
Insert Data In relation Field instead of updating

### DISCUSSION THREAD

**Discussion by manishonc** [pocketbase/pocketbase#3038] [Q&A]
How do I insert New Data in the Relation Field instead of updating the whole relation 

Like Users have Post -

User - related - post.

But when I create a new Post from the frontend I have to  insert a post id in the Users post relation.

Maybe my approach is wrong if there is another solution please guide me.

Thankyou @ganigeorgiev

**Comment by ganigeorgiev** [comment]
I don't understand what do you mean.

If not already, you could check https://pocketbase.io/docs/working-with-relations/
